darwin/Platform.c

```diff
@@ -290,6 +290,24 @@ double Platform_setCPUValues(Meter* mtr, unsigned int cpu) {
    return CLAMP(total, 0.0, 100.0);
 }
 
+#if defined(__arm64__)
+void Platform_setMemoryValues(Meter *mtr) {
+   const DarwinMachine *dhost = (const DarwinMachine *) mtr->host;
+   const struct vm_statistics64 *vm = &dhost->vm_stats64;
+   double page_K = (double) vm_page_size / (double) 1024;
+
+   mtr->total = dhost->host_info.max_mem / 1024;
+   int used = vm->active_count + vm->inactive_count +
```

Comment:
Is it intentional to use signed `int` type here? Note that `vm->active_count` and similar members are in "`natural_t`" type, which is unsigned.